tpot/tpot.py

```diff
@@ -738,6 +740,50 @@ def _select_kbest(self, input_df, k):
         mask_cols = list(training_features.iloc[:, mask].columns) + ['guess', 'class', 'group']
         return input_df[mask_cols].copy()
 
+
+    def _select_fwe(self, input_df, alpha):
+        """ BASED ON THE _select_kbest METHOD
+
+        Uses Scikit-learn's SelectFwe feature selection to filter the subset of features
+           according to p-values corresponding to Family-wise error rate
+        Parameters
+        ----------
+        input_df: pandas.DataFrame {n_samples, n_features+['class', 'group', 'guess']}
+            Input DataFrame to perform feature selection on
+        k: float, optional
+            The highest uncorrected p-value for features to keep
+
+        Returns
+        -------
+        subsetted_df: pandas.DataFrame {n_samples, n_filtered_features + ['guess', 'group', 'class']}
+            Returns a DataFrame containing the 'best' features
+
+        """
+        training_features = input_df.loc[input_df['group'] == 'training'].drop(['class', 'group', 'guess'], axis=1)
+        training_class_vals = input_df.loc[input_df['group'] == 'training', 'class'].values
+        alpha = float(alpha) # force float
```

Comment:
No need to force it to a float. We document it as requiring a float, so if the user passes an invalid value, it should error out on its own.